src/server/services/BruincastServices.js

```diff
@@ -135,22 +125,16 @@ class BruincastServices {
   }
 
   static async getCastListings(term) {
-    const termMedia = await MediaQuery.getMediaForTerm('bruincastmedia', term);
+    const termMedia = await MediaQuery.getMediaForTerm(
+      process.env.DB_COLLECTION_BRUINCAST,
```

Comment:
@maxpluez Wait, should the collection names map defined in constants.js be used here? Or is there a reason the `.env` variable is being used?